parsec/backend/user.py

```diff
@@ -530,6 +534,8 @@ async def _api_user_create(self, client_ctx, msg):
 
         except UserAlreadyExistsError as exc:
             return {"status": "already_exists", "reason": str(exc)}
+        except UserLimitReached:
+            return {"status": "not_allowed", "reason": "User's limit reached"}
```

Comment:
`User's limit reached` ? shouldn't it be `Users limit reached` ?

poke @Max-7 who knows how to spell "retreive" :trollface: